content/F2016/F16-Office-Hours.md

```diff
@@ -3,7 +3,10 @@ Date: 2016-10-06 19:00
 Category: Blog
 Tags: Office Hours
 Author: Rolina Wu
-Summary: This is the regular Office hours for Fall 2016
+Summary: Announcing our Office Hours for Fall 2016
+
+Come visit us during our Office hours or just swing by to say hi! We have various technical interview resources and publications available for you to read during Office Hours. There's always an officer that will be available in the office if you have any questions or want to meet some new faces. :)
+You can also read more about what our office provides on our [contact us](https://wics-site-pr-267.herokuapp.com/contact-us/) page.
```

Comment:
this link is temporary actually - it should be `{filename}/pages/contact-us/`  I think and that will work with any base url